src/test/java/com/redhat/devtools/intellij/kubernetes/model/resource/KubernetesVersionFactoryTest.java

```diff
@@ -37,16 +37,16 @@ public void create_should_create_kubernetes_version() {
     @Test
     public void create_should_create_non_kubernetes_version() {
         // given
-        Version version = KubernetesVersionFactory.create("darthVader");
+        KubernetesVersionFactory.Version version = KubernetesVersionFactory.create("darthVader");
```

Comment:
the same as above for all occurrences in this test class.

Reply:
good catch!